draft-ietf-masque-h3-datagram.md

```diff
@@ -249,23 +141,29 @@ corresponding stream.
 # Capsules {#capsule}
 
 This specification introduces the Capsule Protocol. The Capsule Protocol is a
-sequence of type-length-value tuples that allows endpoints to reliably
-communicate request-related information end-to-end, even in the presence of
-HTTP intermediaries.
-
-
-## Capsule Protocol {#capsule-protocol}
+sequence of type-length-value tuples that new HTTP Methods or new HTTP Upgrade
+Tokens can choose to use. It allows endpoints to reliably communicate
+request-related information end-to-end on HTTP request streams, even in the
+presence of HTTP intermediaries. The Capsule Protocol can be used to exchange
+HTTP Datagrams over versions of HTTP that do not support the QUIC DATAGRAM frame.
```

Comment:
ambiguity alert: do we striclty mean HTTP versions? Presumably HTTP/3 connections that do not support QUIC DATAGRAM (or chose not to use it even if supported) are also in scope.

Consider moving the last sentence here to para 3, so that mention of Capsule Protocol plus HTTP Datagrams is grouped.

Reply:
Fixed